### ISSUE TITLE
Typescript: `<Ditto />` component doesn't accept a `children` function.

### ISSUE THREAD

**Issue by cool-acid-playbook** [dittowords/ditto-react#7]
Trying to use a children render function it will throw a Typescript error:
```
Type '{ children: ({ text }: { text: string; }) => Element; componentId: string; }' is not assignable to type '(IntrinsicAttributes & DittoProjectProps) | (IntrinsicAttributes & DittoComponentLibraryProps)'.
  Property 'children' does not exist on type '(IntrinsicAttributes & DittoProjectProps) | (IntrinsicAttributes & DittoComponentLibraryProps)'.
```
Using `ditto-react@0.2.0`.

**Comment by azjgard** [comment]
Hi @cool-acid-playbook, this has been fixed in the v0.3.0 release which is now live on npm!

You can either use the `<Ditto />` or `<DittoComponent />` components - the latter has better type safety and inference:
```jsx
import { Ditto } from 'ditto-react';

// 'text' must be explicitly typed
<Ditto componentId="excellent.validation">
  {(text: string) => <>{text}</>}
</Ditto>
```
```jsx
import { DittoComponent } from 'ditto-react';

// type of 'text' is inferred
<DittoComponent componentId="excellent.validation">
  {(text) => <>{text}</>}
</DittoComponent>

**Comment by cool-acid-playbook** [comment]
Thanks! I will give it a try!
